scaffolder-templates/basic-workflow/.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,44 @@
+# Requirement: add a K8S_SECRET secret to your organization with the SA token
+# oc get secret backstage-k8s-token-4l5xv -n backstage-system -oyaml > secret.yaml
+# gh secret --repo parodos-dev/test-swf set K8S_SECRET <secret.yaml 
```

Comment:
Should we specify here the requirement for the SA token (e.g. which RBAC is required)?

Reply:
adding a generic `cluster-admin`, but we should review this statement and reduce the permissions to the minimum

Reply:
Added https://github.com/parodos-dev/workflow-software-templates/issues/20 to track the request